lib/rucio/transfertool/fts3.py

```diff
@@ -576,8 +584,16 @@ def initialize(self, session, logger=logging.log):
             elif file_state == FTS_STATE.FAILED and job_state == FTS_STATE.FAILED or \
                     file_state == FTS_STATE.FAILED and not self._multi_sources:  # for multi-source transfers we must wait for the job to be in a final state
                 if self._is_recoverable_fts_overwrite_error(self.request(session), reason, self._file_metadata):
+                    file_meta = self._file_metadata
+                    self.logger(logging.DEBUG,
```

Comment:
I'm not sure this log is needed at all, even in debug. I don't think it bring additional information compared to existing log messages. But I will not fight strongly against it.